VMBackup/main/handle.py

```diff
@@ -69,12 +70,16 @@ def main():
         snapshot_info_array = None
         total_used_size = 0
         size_calculation_failed = False
+        eventlogger = None
         HandlerUtil.waagent.LoggerInit('/dev/console','/dev/stdout')
         hutil = HandlerUtil.HandlerUtility(HandlerUtil.waagent.Log, HandlerUtil.waagent.Error, CommonVariables.extension_name)
         backup_logger = Backuplogger(hutil)
         MyPatching, patch_class_name, orig_distro = GetMyPatching(backup_logger)
         hutil.patching = MyPatching
         configSeqNo = -1
+        hutil.try_parse_context(configSeqNo)
```

Comment:
this looks unrelated to logging change. why is this required?

Reply:
To get/fetch the evnets folder mainly